mypy/server/deps.py

```diff
@@ -101,14 +154,99 @@ def visit_import(self, o: Import) -> None:
     def visit_import_from(self, o: ImportFrom) -> None:
         assert o.relative == 0  # Relative imports not supported
         for name, as_name in o.names:
-            assert as_name is None or as_name == name
             self.add_dependency(make_trigger(o.id + '.' + name))
 
+    def visit_block(self, o: Block) -> None:
+        if not o.is_unreachable:
+            super().visit_block(o)
+
     def visit_assignment_stmt(self, o: AssignmentStmt) -> None:
```

Comment:
PS. Traverser should also visit statement's .analyzed attribute.

Reply:
Turns out that `AssignmentStmt` doesn't have `.analyzed`.